contracts/zksync/deploy.sh

```diff
@@ -10,15 +10,17 @@ if [ -z "$PAYMENT_REGISTRY_PROXY_ADDRESS" ]; then
 fi
 
 DEPLOY="deploy"
-if [ "$TEST" == true ]; then
+if [ "$TEST" = "true" ]; then
```

Comment:
Can't this just be if [ $TEST ]; then

Reply:
I think it is this way because TEST is a string not a boolean

Reply:
exactly, 
https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script